keyboards/keyap/keymaps/via/keymap.c

```diff
@@ -0,0 +1,60 @@
+// Copyright 2023 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+#define LED1 B0
+#define LED2 B1
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /*
+     * ┌───┬───┬───┐
+     * │ A │ B │ C │
+     * ├───┼───┼───┤
+     * │ D │ E │ F │
+     * ├───┼───┼───┤
+     * │ G │ H │ I │
+     * └───┴───┴───┘
+     */
+    [0] = LAYOUT_ortho_3x3(KC_A, KC_B, KC_C,
+                           KC_P, KC_E, KC_D,
+                           KC_G, KC_H, KC_I)
+    };
+
+const uint16_t PROGMEM encoder_map[][1][2] = {[0] = {{KC_C, KC_D}}};
```

Comment:
```suggestion
const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][NUM_DIRECTIONS] = {
    [0] = {ENCODER_CCW_CW(KC_C, KC_D)}
};
```

Reply:
looool if i do that my build fails

Reply:
This is literally copied from the default keymap.

Reply:
but this does not work as should

Reply:
I can make NUM_ENCODERS NUM_DIRECTIONS as defines to keep the standarts of embedded sw development and not to have magic numbers in code